docs/api/reference/linea-estimategas.mdx

```diff
@@ -177,3 +182,42 @@ decimals to get the wei value. You can use any hexadecimal to decimal converter
 [RapidTables](https://www.rapidtables.com/convert/number/hex-to-decimal.html).
 
 :::
+
+## Override state values
+
+You can override an account with temporary state values before making the call. This allows you to
+make temporary state changes without affecting the actual blockchain state.
+
+The following example estimates the cost for transferring an ERC20 token for an account which does not
```

Comment:
```suggestion
The following example estimates the cost for transferring an ERC-20 token for an account which does not
```

Per the [Consensys Editorial Style Guide](https://www.notion.so/consensys/Consensys-Editorial-Style-Guide-d5b9867e85df4ae38f8bed44f61a77d5?pvs=4). Though we're not consistent with this throughout this site so I'll make an issue to fix.